roles/openshift_control_plane/tasks/upgrade.yml

```diff
@@ -142,6 +142,16 @@
     state: absent
   when: not openshift_is_atomic | bool
 
+- name: Backup atomic-openshift systemd services
+  copy:
+    src: "{{ item }}"
+    dest: "{{ item | regex_replace('[.]', '-') }}.bak-{{ ansible_date_time.iso8601_basic_short }}"
```

Comment:
`backup: true` option of `copy` task would add standard ansible timestamp instead

Reply:
I could not think of a clean way to use `backup: true` because it only creates a backup file if the `dest` is being overwritten, which is not the case here.

Reply:
Oh, correct, `backup` won't work if the file didn't change